### PR TITLE
Add service unavailable banner

### PR DESCRIPTION
**Pull request by tombye**
Add a notification banner for services no longer available to buy.

https://www.pivotaltracker.com/story/show/110062348

image

Note: this is a re-vamped version of https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/210

### CODE REVIEW

File: toolkit/scss/_notification-banners.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 @import "_colours.scss";
 @import "_digital-marketplace-colours.scss";
 @import "_typography.scss";
+@import "_measurements.scss";
+
+// Required placeholders:
+// @import "placeholders/_temporary-messages.scss";
```
Comment (quis) [inline toolkit/scss/_notification-banners.scss]: This reads like an import that’s been commented out. Might read better without the `@import`?

File: toolkit/scss/_temporary-message.scss [hunk 2]
```diff
@@ -4,16 +4,21 @@
 @import "_shims.scss";
 @import "_conditionals.scss";
 
-%temporary-message,
+// Required placeholders:
+// @import "placeholders/_temporary-messages.scss";
```
Comment (quis) [inline toolkit/scss/_temporary-message.scss]: Same here.

**Comment by tombye** [comment]
I've removed the `@import` from the comments and rewrote them to be clearer (in aa44859)

**Comment by quis** [comment]
LGTM now, excellent perseverance :+1:

**Pull request merged**